src/tensorlake/functions_sdk/functions.py

```diff
@@ -53,23 +55,18 @@ def is_pydantic_model_from_annotation(type_annotation):
     return False
 
 
-class PlacementConstraints(BaseModel):
-    min_python_version: Optional[str] = "3.9"
-    max_python_version: Optional[str] = None
-    platform: Optional[str] = None
-    image_name: Optional[str] = None
-
-
 class TensorlakeCompute:
     name: str = ""
     description: str = ""
     image: Optional[Image] = None
     secrets: Optional[List[str]] = None
-    placement_constraints: List[PlacementConstraints] = []
     accumulate: Optional[Type[Any]] = None
     input_encoder: Optional[str] = "cloudpickle"
     output_encoder: Optional[str] = "cloudpickle"
     inject_ctx = False
+    retries: Optional[Retries] = None  # Use graph retry policy if not set
+    timeout: int = 300  # 5 minutes
+    resources: Resources = Resources()
```

Comment:
All the resource constraints have default values, please see the Resources constructor.